morecantile/models.py

```diff
@@ -925,3 +930,154 @@ def quadkey_to_tile(self, qk: str) -> Tile:
                 raise QuadKeyError("Unexpected quadkey digit: %r", digit)
 
         return Tile(xtile, ytile, i + 1)
+
+    def extrema(self, zoom: int) -> Dict:
+        """Return TileMatrix Extrema.
+
+        Note: Equivalent of `mercantile.minmax`
+
+        Parameters
+        ----------
+        zoom : int
+            The zoom level.
+
+        Returns
+        -------
+        Dict
+
+        """
+        m = self.matrix(zoom)
+        return {
+            "x": {"min": 0, "max": m.matrixWidth - 1},
+            "y": {"min": 0, "max": m.matrixHeight - 1},
+        }
```

Comment:
this is like the `minmax` method of mercantile but some TMS can have diff min/max for each axis